tests/snuba/search/test_backend.py

```diff
@@ -2287,7 +2293,7 @@ def test_first_release_any_or_no_environments(self):
         #         env=[production]: nothing
         #
         #     first_release: 2
-        #         env=[]: A, C
+        #         env=[]: C
```

Comment:
Since A was first seen in a different release, it shouldn't be included here. AFAICT this should capture the updated behavior